config/prow/cluster/monitoring/mixins/lib/config.libsonnet

```diff
@@ -38,5 +38,32 @@
     heartbeatJobs: [
       {name: 'ci-test-infra-prow-checkconfig', interval: '9m', alertInterval: '20m'},
     ],
+
+    // Additional scraping endpoints
+    additionalScraping: [
```

Comment:
These are not additional scrape endpoints, but rather targets for the blackbox prober. `probeTargets` would be more accurate.

Reply:
fixed